addons/point_of_sale/static/src/app/components/inputs/input/input.js

```diff
@@ -28,6 +28,7 @@ export class Input extends TModelInput {
         class: { type: String, optional: true },
         callback: { type: Function, optional: true },
         isOpenCallback: { type: Function, optional: true },
+        readOnly: { type: Boolean, optional: true },
```

Comment:
```suggestion
        readonly: { type: Boolean, optional: true },
```

Use the same naming as default html one.